xrpl4j-core/src/test/java/org/xrpl/xrpl4j/model/transactions/json/EscrowJsonTests.java

```diff
@@ -73,7 +73,6 @@ public void testEscrowCreateJson() throws JsonProcessingException, JSONException
       "    \"DestinationTag\": 23480,\n" +
       "    \"SourceTag\": 11747,\n" +
       "    \"Sequence\": 1,\n" +
-      "    \"Flags\": 2147483648,\n" +
```

Comment:
Double-checking we have variants that ensure this works with existing flags? (not just empty?)

Reply:
You cannot set `flags` on `Escrow` transactions. Before this change, the `flags` field would always be `tfFullyCanonicalSig` -- after this change, they will always be empty, so will not show up in JSON.

Reply:
Per @nkramer44, once we make all the Flags fields `@Default` this will be testable again.